new/LIT6942SalamLakiGalila.xml

```diff
@@ -0,0 +1,71 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6942SalamLakiGalila" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Salām lakki ʾo-hagara Galilā</title>
```

Comment:
`en`

Reply:
You mean, I shall add an English translation to the Geez-title?

Reply:
I mean the attribute XML:lang refers to the text of the tag, not the text of the work. The phrase you provide is in English so it should be en instead of gez.

Reply:
Sorry was confused, was referring to another record ) here gez is correct but gez stands for Fidel, transcribed text should have type normalized.